roles/edpm_derive_pci_device_spec/tasks/derive_pci.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Copyright 2024 Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+- name: Check valid input for edpm_derive_sriov_device_spec_list
+  ansible.builtin.fail:
+    msg: "List of sriov device_spec cannot be empty - edpm_derive_sriov_device_spec_list"
+  when: edpm_derive_sriov_device_spec_list | length > 0
```

Comment:
@vcandapp Need to update condition length==0 for fail message.

Reply:
Done